src/tasks/netSdk/netSdkTaskUtils.ts

```diff
@@ -61,18 +56,17 @@ export async function getNetSdkBuildCommand(context: DockerTaskExecutionContext)
 
 export async function getNetSdkRunCommand(context: DockerTaskExecutionContext): Promise<string> {
     const client = await ext.runtimeManager.getClient();
-    const folderName = await quickPickWorkspaceFolder(context.actionContext, ErrMsgNoWorkplaceFolder);
 
     const command = await client.runContainer({
         detached: true,
         publishAllPorts: true,
-        name: getDefaultContainerName(folderName.name, NetSdkDefaultImageTag),
+        name: getDefaultContainerName(context.folder.name, NetSdkDefaultImageTag),
         environmentVariables: {},
         removeOnExit: true,
-        imageRef: getDefaultImageName(folderName.name, NetSdkDefaultImageTag),
+        imageRef: getDefaultImageName(context.folder.name, NetSdkDefaultImageTag),
         labels: defaultVsCodeLabels,
         mounts: await getRemoteDebuggerMount(),
-        exposePorts: [8080], // hard coded for now since the default port is 8080
+        exposePorts: [8080, 80], // the default port is 8080 for .NET 8 and 80 for .NET 7
```

Comment:
We should consider only exposing ports if the project is targeting web/Asp.NET and it doesn't already expose ports using MSBuild config.